app/controllers/api/braintree_controller.rb

```diff
@@ -13,6 +13,28 @@ def transaction
     end
   end
 
+  def webhook
+    webhook_notification = Braintree::WebhookNotification.parse(params[:bt_signature], params[:bt_payload])
+    if webhook_notification.kind == Braintree::WebhookNotification::Kind::SubscriptionChargedSuccessfully
+      card_num = webhook_notification.subscription.transactions.last.credit_card_details.last_4
+      query_params = {
+          is_subscription: true,
+          email: webhook_notification.subscription.transactions.last.customer_details.email,
+          card_num: card_num.nil? ? 'PYPL' : card_num,
```

Comment:
I don't understand the question? `ChampaignQueue` isn't available in this scope and shouldn't have a `card_num` property available.